### PR TITLE
Don't show team options when you aren’t on a team subscription

### PR DESCRIPTION
**Pull request by jasonzurita**
Resolves https://github.com/pointfreeco/pointfreeco/issues/297

rr @mbrandonw

### CODE REVIEW

File: Sources/PointFree/Account/Account.swift [hunk 1]
```diff
@@ -767,4 +767,9 @@ private struct AccountData {
   var isSubscriptionOwner: Bool {
     return self.currentUser.id == self.subscriptionOwner?.id
   }
+
+  var isTeamSubscription: Bool {
+    guard let id = stripeSubscription?.plan.id else { return false }
+    return id == Tagged.teamMonthly || id == Tagged.teamYearly
```
Comment (mbrandonw) [inline Sources/PointFree/Account/Account.swift]: is it possible to drop the `Tagged` type from these?

File: Sources/PointFreeTestSupport/PointFreeTestSupport.swift [hunk 2]
```diff
@@ -26,6 +26,10 @@ extension Environment {
     |> (\Environment.database.fetchSubscriptionTeammatesByOwnerId) .~ const(pure([.mock]))
     |> \.database.fetchTeamInvites .~ const(pure([.mock]))
     |> \.stripe.fetchSubscription .~ const(pure(.teamYearly))
+
+  public static let individualMonthly = mock
+    |> (\Environment.database.fetchSubscriptionTeammatesByOwnerId) .~ const(pure([.mock]))
```
Comment (mbrandonw) [inline Sources/PointFreeTestSupport/PointFreeTestSupport.swift]: you should be able to drop the `Environment` from this key path.

File: Sources/PointFree/Account/Account.swift [hunk 3]
```diff
@@ -767,4 +767,9 @@ private struct AccountData {
   var isSubscriptionOwner: Bool {
     return self.currentUser.id == self.subscriptionOwner?.id
   }
+
+  var isTeamSubscription: Bool {
+    guard let id = stripeSubscription?.plan.id else { return false }
```
Comment (mbrandonw) [inline Sources/PointFree/Account/Account.swift]: can you use `self.stripeSubscription`? we like explicit `self`s.

File: Sources/PointFree/Account/Account.swift (hunk 1 above)
Comment (jasonzurita) [reply Sources/PointFree/Account/Account.swift]: yes! I ran into some issues with the type checker with an earlier version, and I accidentally left them in :(.

File: Sources/PointFree/Account/Account.swift (hunk 3 above)
Comment (mbrandonw) [reply Sources/PointFree/Account/Account.swift]: we like the distinction between local/global/instance vars more than being explicit about capture. and there's not a single class in the entire Point-Free code base so no risk of capture!

Comment (jasonzurita) [reply Sources/PointFree/Account/Account.swift]: ahh, got ya — thanks, and the beauty of not being tied to Cocoa/Cocoa Touch!

**Comment by mbrandonw** [comment]
Awesome, this looks great! Gonna merge and deploy now...

**Pull request merged**